.travis.yml

```diff
@@ -166,6 +163,12 @@ script:
           gdb -return-child-result -batch -ex r -ex bt --args python $PYTHON_ARGS tests.py $NOSE_ARGS $TEST_ARGS
         fi
       else
+        # Workaround for pytest-xdist flaky colletion order
+        # https://github.com/pytest-dev/pytest/issues/920
+        # https://github.com/pytest-dev/pytest/issues/1075
+        export PYTHONHASHSEED=$(shuf -i 1-4294967295 -n 1)
+        echo PYTHONHASHSEED=$PYTHONHASHSEED
```

Comment:
Aren't the tests that really care about this run in a shell-call with their own seeds set?